nestkernel/simulation_manager.cpp

```diff
@@ -813,27 +842,6 @@ nest::SimulationManager::update_()
 }
```

Comment:
I see now -- but this would also be a problem for multiple simulate() calls. But this may break the ideas that `simulate(n) = prepare(); run(n/2); run(n/2); cleanup()`
I think @sdiazpier is the only person who can answer this, or give a good suggestion on what we should do.